arches/app/models/migrations/11799_require_graph_slug.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 5.2b1 on 2025-03-19 11:14
+
+import django.core.validators
+import re
+from django.db import migrations, models
+from slugify import slugify
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("models", "11882_principal_user_read_only"),
+    ]
+
+    def add_missing_graph_slug(apps, schema_editor):
+        Graph = apps.get_model("models", "GraphModel")
```

Comment:
I'd use either Graph or GraphModel but not mix the two (this is importing GraphModel but calling it Graph) -- I guess it should be Graph on both sides to get Graph.save() validation checks.